iidm/iidm-api/src/main/java/com/powsybl/iidm/network/LoadingLimits.java

```diff
@@ -80,4 +82,13 @@ interface TemporaryLimit {
      * @return the temporary limit value or NaN if there is no temporary limit for this acceptable duration
      */
     double getTemporaryLimitValue(int acceptableDuration);
+
+    /**
+     * Set the temporary limit value. Throws an exception when the given acceptable duration is not valid, changes the value but
+     * logs a warning when the new value is not valid.
```

Comment:
Maybe something like this will be clearer?
```suggestion
     * Set the temporary limit value. 
     * <p>Throws an exception when no temporary limit of the given acceptable duration is found,
     * and changes the value but logs a warning when the new value is not valid.</p>
```